.github/workflows/deploy.yml

```diff
@@ -96,7 +96,7 @@ jobs:
         run: |
           bundle exec rake db:create
           bundle exec rails db:schema:load
-          bundle exec rspec --format progress
+          bundle exec rspec --format progress --exclude-pattern "spec/smoke_tests/**/*_spec.rb"
```

Comment:
Does this mean the smoke tests will execute when we run RSpec locally via:

```
bundle exec rspec
```

?

Since these are a different class of test - based on the assumption that the server is up & running - I wonder if that makes our DX a little bit worse.

Perhaps we could do [something like this](https://stackoverflow.com/a/53661752) so that smoke tests are excluded by default, and then a different command has to be used to specifically run them?

Reply:
Nice! I like it!